include/openthread/thread.h

```diff
@@ -1114,6 +1114,28 @@ void otThreadSetStoreFrameCounterAhead(otInstance *aInstance, uint32_t aStoreFra
  */
 uint32_t otThreadGetStoreFrameCounterAhead(otInstance *aInstance);
 
+/**
+ * Attempts to attach a Wake-up End Device.
+ *
+ * Requires `OPENTHREAD_CONFIG_WAKEUP_COORDINATOR_ENABLE` to be enabled.
+ *
+ * The attachment starts with transmitting a wake-up frame sequence to the Wake-up End Device.
+ * During the wake-up sequence, and for a short time after the last wake-up frame is sent, the Wake-up Coordinator keeps
+ * its receiver on to be able to receive an initial mesh link establishment message from the WED.
+ *
+ * @param[in] aInstance         A pointer to an OpenThread instance.
+ * @param[in] aWedAddress       The extended address of the Wake-up End Device.
+ * @param[in] aWakeupIntervalUs An interval between consecutive wake-up frames (in microseconds).
+ * @param[in] aWakeupDurationMs Duration of the wake-up sequence (in milliseconds).
+ *
+ * @retval OT_ERROR_NONE          Successfully started the attachment.
+ * @retval OT_ERROR_INVALID_STATE Another attachment request is still in progress.
```

Comment:
Should this API returns `OT_ERROR_INVALID_ARGS` when `aWakeupIntervalUs == aWakeupDurationMs  == 0`?